src/constants.ts

```diff
@@ -9,6 +9,11 @@ export const GAS_LIMIT_PERCENTAGE_ETH = 120
 export const MAX_UINT = MaxUint256
 export const ZERO_NUMBER = BigNumber.from('0')
 
+export enum ChainId {
```

Comment:
I think it makes sense to move it over to `abis` as we can use it across our Front-end repos. This file should ideally contain the constants that are used purely within this package